### DISCUSSION TITLE
1.0.9 버전으로 창 하나만 켜도 블락 되는거 같습니다

### DISCUSSION THREAD

**Discussion by gingno** [changdoc/naver-vaccine-macro#51] [Q&A]
어제는 사용 잘 하고 있었는데 창 하나만 키고 구동 했는데도 블락 당했는데 제가 뭘 잘못한건지 ㅜㅜ 다른 분들은 어떠신가해서요

**Comment by changdoc** [comment]
테스트 해보고 있는데 차단되지 않는거 보면 동작에 이상이 좀 있는가보네요.

차단 해제 되면 개발자 도구 console 여시고 interval이 2000으로 잘 동작 중인지 확인 한번 해보세요.
![image](https://user-images.githubusercontent.com/1879494/126578557-16c78b9b-a62e-4705-ba00-b31a2e4f50c0.png)
